src/01-functions.js

```diff
@@ -112,7 +124,13 @@ function averageThree() {}
  *  compareTypes("left", 10);
  *  //> false
  */
-function compareTypes() {}
+function compareTypes(a,b) {
+  if (typeof a === typeof b){
```

Comment:
Could just return this condition`typeof a === typeof b`. Also as a stretch try to think of some edge cases where this would not work and rewrite the code to cover those cases.